File: Tema 2 - Recogida de datos por teclado mediante formularios/Ejercicios/ejercicio9 volumen de un cono/archives/9.py
import sys
import math

radio = float(sys.argv[1])
altura = float(sys.argv[2])
total = (math.pi * pow(radio, 2) * altura) / 3


mensaje = (f"""Vol = (&#928*radio^2*altura)/3
			<br><br>radio = {radio}
			<br><br>altura = {altura}
			<br> <br> Volumen= {round(total, 2)} cm^3""")

print(f" {mensaje}")

File: Tema 2 - Recogida de datos por teclado mediante formularios/Ejercicios/ejercicio9 volumen de un cono/archives/9.php
<?php
$radio = $_POST["num1"];
$altura = $_POST["num2"];
$total = (pi() * pow($radio, 2) * $altura) / 3;

$mensaje = "Vol=( &#928*radio²*altura)/3" .
	"<br><br>Volumen = $radio " .
	"<br><br>Volumen = $altura " .
	"<br><br>Volumen = " . round($total, 2) . "cm³";

echo $mensaje;
	
	

















/* if ($metodo == "php") { */
	// Sumar utilizando PHP
/* 	$resultado = $num1 * $num2; */
/* 	echo "la multiplicacion de <br> $num1 * $num2 es: ".$resultado;
} elseif ($metodo == "python") { */
	// Sumar utilizando Python
/* 	$output = shell_exec("python 1.py " . $num1 . " " . $num2);
	echo $output;
} else {
	echo "Método no válido";
} */
